ArmoniK.Extension.CSharp.DllWorker/Program.cs

```diff
@@ -0,0 +1,21 @@
+// This file is part of the ArmoniK project
+// 
+// Copyright (C) ANEO, 2021-2024. All rights reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License")
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using ArmoniK.Api.Worker.Utils;
+using ArmoniK.Extension.CSharp.DllWorker;
+
+WorkerServer.Create<ComputerService>(serviceConfigurator: collection => collection.AddSingleton<ServiceRequestContext>())
```

Comment:
Update to this https://github.com/aneoconsulting/ArmoniK.Extensions.Csharp/commit/da71390b29871c9ea90ea73d6cbfc496b4b0ea84#diff-2c1951031ad916bd06826210c181ed661616f413820d59008143bcc156bf86feR23-R27